event_handler_publisher_delay_logging.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * This event handler can be used in conjunction with a "Point value update"
+ * detector set to "Do not log" (to avoid creating a bajillion events)
+ * to log the difference between the point value timestamp from the
+ * edge and the actual time the value was received & processed in the cloud.
+*/
+
+const Common = Java.type('com.serotonin.m2m2.Common');
+const tagNamesToLog = ['site', 'deviceId'];
+
+/*
+ * Called when the Mango event is raised.
+ * You must implement this method.
+ *
+ * @param event
+ */
+function eventRaised(event) {
+    try{
+        //Confirm this eventType is a data point
+        const currEventType = event.getEventType();
+        if (currEventType.toString().substring(0,18) == 'DataPointEventType') {

```

Comment:
`===` is preferred, I think this will work despite the fact we are comparing a Java string to a JavaScript string.